src/main/scala/com/fortysevendeg/android/sqlite/SQLDroidConnection.scala

```diff
@@ -0,0 +1,220 @@
+package com.fortysevendeg.android.sqlite
+
+import java.sql._
+import java.util.Properties
+import java.util.concurrent.Executor
+
+import com.fortysevendeg.android.sqlite.logging.{AndroidLogWrapper, LogWrapper}
+import com.fortysevendeg.android.sqlite.metadata.SQLDroidDatabaseMetaData
+import org.sqldroid.{SQLDroidPreparedStatement, SQLDroidStatement}
+
+class SQLDroidConnection(
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Number of methods in class exceeds 30](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/809217331/issues/source?bid=2442814&fileBranchId=2738989#l11)